packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageHeader.tsx

```diff
@@ -149,7 +150,8 @@ export function TracePageHeaderFn(props: TracePageHeaderEmbedProps & { forwarded
       return { ...rest, value: renderer(trace) };
     });
 
-  const traceShortID = trace.traceID.slice(0, 7);
+  const traceIDLength = getConfigValue('traceIDLength');
+  const traceShortID = trace.traceID.slice(0, traceIDLength);
```

Comment:
This doesn't play nice with styling when very long.

<img width="1168" alt="image" src="https://github.com/user-attachments/assets/6967e949-de58-4c17-9653-ad794fc55c65" />

I think the ID needs to be in smaller font.